src/frontends/paddle/tests/test_models/gen_scripts/generate_multi_tensor_split.py

```diff
@@ -35,11 +40,7 @@ def create_multi_output_model():
     inp_dict = {'x': inp_blob_1}
     res_paddle = exe.run(fluid.default_main_program(), fetch_list=var, feed=inp_dict)
 
-    paddle.fluid.io.save_inference_model(os.path.join(sys.argv[1], "multi_tensor_split"),
-                                  list(inp_dict.keys()), var, exe,
-                                  model_filename="multi_tensor_split.pdmodel",
-                                  params_filename="multi_tensor_split.pdiparams")
-
+    paddle.static.io.save_inference_model(os.path.join(sys.argv[1], "multi_tensor_split", "multi_tensor_split"), [x], var, exe)
```

Comment:
Let's use the saveModel api as other gen_scripts.

Reply:
done https://github.com/openvinotoolkit/openvino/pull/23010/commits/38c1e6d4be184c5c7522fe1a65f17b6b6f6bc2b6